src/app/pages/feed-overview-page/feed-overview-page.component.html

```diff
@@ -21,7 +21,7 @@ <h2 class="feed-overview_title">{{feed.name}}</h2>
         <div class="newfeeds">
```

Comment:
refactor to .feed-overview_new-feeds
change .feed-overview_content to just .feed_overview

Reply:
.feed-overview_content is in line 1